### DISCUSSION TITLE
late payments with approval

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3754] [Q&A]
Hello I have a task of a product that requires approval in order to be paid.

How can I accept payments after the last date of payment? If the pretix administrator is late to give the approval and the last date of payment expires, then the customer gets a ticket link where there is no payment link in order to pay for the order.

Thank you in advance.

**Comment by raphaelm** [comment]
You can't use the last day of payments in that scenario, there is no override for it

**Comment by alemao8** [comment]
ok I see.
Is there another way to bypass the expiration of payments mentioned above in our scenario ?
